package.json

```diff
@@ -17,7 +17,7 @@
     "copy-all": "npm run copy-scss && npm run copy-resources && npm run copy-images",
     "copy-scss": "copyfiles -u 1 src/**/*.scss ./dist",
     "copy-resources": "copyfiles -u 1 src/**/Resources.js ./dist",
-    "copy-images": "copyfiles -u 1 src/**/*.png ./dist",
+    "copy-images": "copyfiles -u 1 src/**/*.svg ./dist",
```

Comment:
should we call it asset and keep?
and should we change to 
src/img/** or should we not change?

Reply:
I'm keeping it as-is for now, we can change it to assets later.